Add text prediction to writing assistance (#2291)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

To use **Basic** spelling and grammar checking: 
1. Go to `edge://settings/languages`.
1. In the **Use writing assistance** section, select **Basic**.

To turn off all spelling and grammar capabilities:
1. Go to `edge://settings/languages`.
1. Turn off the **Use writing assistance** toggle.

#### Languages

When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device. 
 
#### Text prediction 

Microsoft Edge automatically provides word and sentence predictions in certain text boxes on the web. These predictions are only visible to you and are not inserted into the text box until you press `Tab` or the `Right arrow` key to accept them.

To turn off text predictions in Microsoft Edge: 

1. Go to `edge://settings/languages`.

2. Turn off the **Use text prediction** toggle.

> If the **Use text prediction** toggle is turned on, Microsoft Edge sends the text in the text box, your top language from the browser setting, and a pseudonymous text box identifier to a Microsoft cloud service over a secure HTTPS connection. The Microsoft cloud service processes the text to generate a relevant text prediction. Typed characters and text predictions are cached for up to 30 days, for service quality and performance improvement purposes only.  

Reviewer: do we use "pseudonymous" elsewhere? I'm not sure what this means in this context? 

maybe, "a text box identifier (not associated with your account)" ?

> If the **Use text prediction** toggle is turned on, Microsoft Edge sends the text in the text box, your top language from the browser setting, and a pseudonymous text box identifier to a Microsoft cloud service over a secure HTTPS connection. The Microsoft cloud service processes the text to generate a relevant text prediction. Typed characters and text predictions are cached for up to 30 days, for service quality and performance improvement purposes only.  

Author: I've added "the text box identifier is not associated with your account", so we explain that none of the data sent for text predictions is identifiable in the first place. Taking this as a follow up for future updates.

> If you are browsing while using **InPrivate** or **Guest** mode, text prediction is turned off. If your device is managed by using group policies set by your organization, the feature may be disabled depending on the administrator's policies of your organization.  

Author: Can it be "editing a password field" rather than "using a password edit field"?

<!-- ====================================================================== -->
## Writing assistance

To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including spell checking, grammar checking, and text prediction. 

> By default, Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.

Reviewer 2: ```suggestion
By default, Microsoft Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.
```

<!-- ====================================================================== -->
## Writing assistance

> To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including enhanced spelling and grammar checking and text prediction. 

Reviewer 2: ```suggestion
To help you write faster and with fewer mistakes on the web, Microsoft Edge provides writing assistance tools, including spell checking, grammar checking, and text prediction. 
```
heading toward making this a more general intro to the section; only 1 of the 3 options is 'enhanced'.

avoiding "x and y and z" construction

> To help you write faster and with fewer mistakes on the web, Microsoft Edge provides different writing assistance tools, including enhanced spelling and grammar checking and text prediction. 

Reviewer 3: Do we need to have the word different.  I think we could eliminate it.

Reviewer 2: The word 'different' was removed in this PR at some point.
Latest:
https://github.com/MicrosoftDocs/edge-developer/pull/2291/files

Please close this comment thread.  @JasonAndrewWriter

> By default, Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.   

Reviewer 2: ```suggestion
By default, Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.
```
trailing spaces are significant in Markdown (manual linebreak)

> To use **Basic** spelling and grammar checking: 

Reviewer 2: ```suggestion
If you select **Basic** instead of **Microsoft Editor**, Microsoft Edge will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services.

To use **Basic** spelling and grammar checking: 
```
Introduce the concept of Basic above the list of steps, and avoid the word 'service' for Basic.

> 1. Go to `edge://settings/languages`    

Reviewer 2: ```suggestion
1. Go to `edge://settings/languages`.
```

> 1. Select the **Basic** setting in **Use writing assitance**

Reviewer 2: ```suggestion
1. In the **Use writing assistance** section, select the **Basic** option button.
```

> If you select **Basic**, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
> 

Reviewer 2: ```suggestion
```
moved to above action steps, to serve as intro

To turn off all spelling and grammar capabilities:

> 1. Go to `edge://settings/languages`

Reviewer 2: ```suggestion
1. Go to `edge://settings/languages`.
```

> 1. Turn off the **Use writing assistance** toggle 

Reviewer 2: ```suggestion
1. Turn off the **Use writing assistance** toggle.
```

> #### Languages

Reviewer 2: ```suggestion
#### Languages

```
According to MarkdownGuide, "You should also put blank lines before and after a heading for compatibility."
https://www.markdownguide.org/basic-syntax/#heading-best-practices
Docs Contributor Guide doesn't explicitly state that, but does use that convention, as does the present repo:
https://raw.githubusercontent.com/MicrosoftDocs/Contribute/main/Contribute/markdown-reference.md

When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device. 
 
#### Text prediction 

> Microsoft Edge will automatically provide word and sentence predictions in certain text boxes on the web. These predictions are only visible to you and are not inserted into the text box until you press Tab or the Right arrow to accept them.  

Reviewer 2: ```suggestion
Microsoft Edge automatically provides word and sentence predictions in certain text boxes on the web. These predictions are only visible to you and are not inserted into the text box until you press `Tab` or the `Right arrow` key to accept them.
```
Per tagging convention for keys in present repo, although repo is not entirely consistent for the Right arrow key.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts - "the Right arrow key".

> To disable text predictions in Microsoft Edge: 

Reviewer 2: ```suggestion
To turn off text predictions in Microsoft Edge: 
```
Privacy article is for general audience as well as technical audience, and this doc'n is about end-user GUI.  Thus lean toward "turn off" instead of "disable".
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/disable-disabled
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/turn-on-turn-off

> 1. Go to `edge://settings/languages`

Reviewer 2: ```suggestion
1. Go to `edge://settings/languages`.
```

> 2. Turn off the **Use text prediction** toggle 

Reviewer 2: ```suggestion
2. Turn off the **Use text prediction** toggle.
```

> If the **Use text prediction** toggle is turned on, Microsoft Edge sends the text in the text box, your top language from the browser setting, and a pseudonymous text box identifier to a Microsoft cloud service over a secure HTTPS connection. The Microsoft cloud service processes the text to generate a relevant text prediction. Typed characters and text predictions are cached for up to 30 days for service quality and performance improvement purposes only.  

Reviewer 2: ```suggestion
If the **Use text prediction** toggle is turned on, Microsoft Edge sends the text in the text box, your top language from the browser setting, and a pseudonymous text box identifier to a Microsoft cloud service over a secure HTTPS connection. The Microsoft cloud service processes the text to generate a relevant text prediction. Typed characters and text predictions are cached for up to 30 days, for service quality and performance improvement purposes only.  
```
imo, comma parses better

> If you are browsing while using **InPrivate** or **Guest** mode, text predictions are turned off. If your device is managed with group policies set by your organization, the feature may be disabled depending on the administrator’s policies of your organization.  

Reviewer 2: ```suggestion
If you are browsing while using **InPrivate** or **Guest** mode, text predictions are turned off. If your device is managed by using group policies set by your organization, the feature may be disabled depending on the administrator's policies of your organization.  
```
Straight instead of curly quotes per style guide or doc contrib guide.

re: 'with' vs. 'by using':
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with

> Text Prediction is only available in select languages and regions. 

Reviewer 2: ```suggestion
Text prediction is only available in select languages and regions.

```
